pandas/core/accessor.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
```

Comment:
Just pushed a commit that does this and adds a comment in the docstring as requested below.  The docstring isn't super-useful at this point because there isn't much here.  After this is merged I'll rebase #17042 and accessor.py will have a coherent theme.